src/lib.rs

```diff
@@ -437,6 +437,28 @@ impl PosixACL {
         }
         Ok(())
     }
+
+    /// Consumes the `PosixACL`, returning the wrapped `acl_t`.
+    /// This can be used directly in FFI calls to the acl library.
+    ///
+    /// To avoid a memory leak, the pointer must either:
+    ///
+    /// - Be converted back to a PosixACL using `PosixACL::from_raw`
+    /// - Have `acl_free` called on it
+    pub fn into_raw(&self) -> acl_sys::acl_t {
```

Comment:
Everywhere else in this file these types are without the `acl_sys::` prefix.

Reply:
Also note the Clippy convention warnings here: https://github.com/intgr/posix-acl/pull/21/checks?check_run_id=501508319